cluster/engine_test.go

```diff
@@ -34,6 +34,7 @@ var (
 		Driver:          "driver-test",
 		KernelVersion:   "1.2.3",
 		OperatingSystem: "golang",
+		OSType:          "linux",
```

Comment:
Can you add a test case for the error case? That Swarm would not be able to build the target image.